scripts/join/join_test.go

```diff
@@ -30,14 +30,15 @@ import (
 
 var (
 	logsFile    = flag.String("logs_file", "join_test.log", "File to write docker logs to")
+	network     = flag.String("network", "omega", "Network to join (default: omega)")
```

Comment:
see `netconf.BindFlag` how to this with proper types.

Reply:
What I saw there is that I'd need to provide a place holder variable to write a value into. But in this test the params need to be declared before `TestJoinNetwork` is called. So I guess I could declare a `var network netconf.ID` and then maybe bind it from the `init` function (I need something running before the test itself), but I remember `init` is discouraged by the linter, so even if I mask it with annotations or use some pre-fixture, I am not sure why this added complexity is better than just reading a string?

Reply:
ok yeah, fair enough, that pattern doesn't really work for these simple global flags, fine to go with this impl